plugin/commentary.vim

```diff
@@ -87,6 +126,12 @@ if !hasmapto('<Plug>Commentary') || maparg('gc','n') ==# ''
   nmap gcc <Plug>CommentaryLine
   nmap cgc <Plug>ChangeCommentary
   nmap gcu <Plug>Commentary<Plug>Commentary
+  xmap gcy   <Plug>CommentaryYank
+  nmap gcy   <Plug>CommentaryYank
+  nmap gcyy  <Plug>CommentaryYankLine
+  xmap gcd   <Plug>CommentaryDupe
+  nmap gcd   <Plug>CommentaryDupe
```

Comment:
Good point. In practice, however, it may not be an issue? In normal 
mode, where the conflict arises, `<Plug>CommentaryDupe` takes a motion, 
and so there is no pause (except if you count Vim waiting for you to 
complete the operator motion). So, you will always be typing either 
`gcdd` or gcd{motion}`. Still, it makes sense to code defensively here 
and wrap the names with parentheses, but that will break the convention 
with the rest of Plug's, so I am going to wait and see if (a) @tpope 
accepts this patch in the first place, and (b) if he amenable to 
modifying all the Plug names.


On 3/19/15 6:03 PM, Justin M. Keyes wrote:
> In plugin/commentary.vim
> <https://github.com/tpope/vim-commentary/pull/42#discussion_r26802462>:
>
>> @@ -87,6 +126,12 @@ if !hasmapto('<Plug>Commentary') || maparg('gc','n') ==# ''
>>    nmap gcc <Plug>CommentaryLine
>>    nmap cgc <Plug>ChangeCommentary
>>    nmap gcu <Plug>Commentary<Plug>Commentary
>> +  xmap gcy   <Plug>CommentaryYank
>> +  nmap gcy   <Plug>CommentaryYank
>> +  nmap gcyy  <Plug>CommentaryYankLine
>> +  xmap gcd   <Plug>CommentaryDupe
>> +  nmap gcd   <Plug>CommentaryDupe
>
> Won't this cause a pause since |<Plug>CommentaryDupeLine| overlaps
> |<Plug>CommentaryDupe|? (solution would be to wrap in parentheses, e.g.,
> |<Plug>(CommentaryDupeLine)| and |<Plug>(CommentaryDupe)|)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tpope/vim-commentary/pull/42/files#r26802462>.
>

-- 



--------------------------------------
Jeet Sukumaran
--------------------------------------
jeetsukumaran@gmail.com
--------------------------------------
Blog/Personal Pages:
    http://jeetworks.org/
GitHub Repositories:
    http://github.com/jeetsukumaran
Photographs (as stream):
    http://www.flickr.com/photos/jeetsukumaran/
Photographs (by galleries):
    http://www.flickr.com/photos/jeetsukumaran/sets/
--------------------------------------